src/w3c/templates/show-people.js

```diff
@@ -1,60 +1,63 @@
     const editorid = p.w3cid ? parseInt(p.w3cid, 10): null;
     const dd = html`<dd class='p-author h-card vcard'
-      property='${re}' resource='${bn}' data-editor-id='${editorid}'></dd>`;
+      data-editor-id='${editorid}'></dd>`;
     const span = conf.doRDFa ?
-      html`<span property='${attr.rp1}' typeof='${attr.rp2}'></span>` :
+      html`<span property='${rp}' typeof='schema:Person'></span>` :
       document.createDocumentFragment();
     const contents = [];
     if (p.url) {
       if (conf.doRDFa) {
-        contents.push(html`<meta property='${attr.rn}' content='${p.name}' />`);
+        contents.push(html`<meta property='schema:name' content='${p.name}' />
+          <a class='u-url url p-name fn'
+          property='schema:url' href='${p.url}'>${p.name}</a>`);
+      } else {
+        contents.push(html`<a class='u-url url p-name fn'
+          href='${p.url}'>${p.name}</a>`);
       }
-      contents.push(html`<a class='u-url url p-name fn'
-        property='${attr.rpu}' href='${p.url}'>${p.name}</a>`);
+    } else if (conf.doRDFa) {
+      contents.push(html`<span property='schema:name' class='p-name fn'>${p.name}</span>`);
     } else {
-      contents.push(html`<span property='${attr.rn}' class='p-name fn'>${p.name}</span>`);
+      contents.push(html`<span class='p-name fn'>${p.name}</span>`);
     }
     if (p.company) {
       if (p.companyURL) {
-        contents.push(html`, <a property='${attr.rwu}'
-          class='p-org org h-org h-card' href='${p.companyURL}'>${p.company}</a>`);
+        if (conf.doRDFa) {
+          contents.push(html`, <span property='schema:worksFor' typeof='schema:Organization'>
+            <meta property='schema:name' content='${p.company}' />
+            <a property='schema:url'
+            class='p-org org h-org h-card' href='${p.companyURL}'>${p.company}</a>
+            </span>`);
+        } else {
+          contents.push(html`, <a class='p-org org h-org h-card' href='${p.companyURL}'>${p.company}</a>`);
+        }
+      }
+      else if (conf.doRDFa) {
+        contents.push(html`, <span property='schema:worksFor' typeof='schema:Organization'>
+          <span property='schema:name'>${p.company}</span>
+          </span>`);
+      } else {
+        contents.push(document.createTextNode(`, ${p.company}`));
       }
-      else contents.push(document.createTextNode(`, ${p.company}`));
     }
     if (p.mailto) {
-      contents.push(html`, <span class='ed_mailto'><a class='u-email email'
-        property='${attr.rm}' href='${`mailto:${p.mailto}`}'>${p.mailto}</a></span>`);
+      if (conf.doRDFa) {
+        contents.push(html`, <span class='ed_mailto'><a class='u-email email'
+          property='foaf:mbox' href='${`mailto:${p.mailto}`}'>${p.mailto}</a></span>`);
+      } else {
+        contents.push(html`, <span class='ed_mailto'><a class='u-email email'
+          href='${`mailto:${p.mailto}`}'>${p.mailto}</a></span>`);
+      }
```

Comment:
Same here

Reply:
We can probably get rid of the `span` too, right? just move the `ed_mailto` into the `<a>` - same above.